content/applications/sales/point_of_sale/payment_methods/terminals/viva_wallet.rst

```diff
@@ -0,0 +1,159 @@
+available in your Viva Wallet account. These credentials include your :ref:`Merchant ID
+<viva_wallet/id-key>`, :ref:`API key <viva_wallet/id-key>`, :ref:`POS APIs credentials
+<viva_wallet/pos-apis>`, and :ref:`Terminal ID <viva_wallet/identifier>` number.
+
+.. _viva_wallet/id-key:
+
+Merchant ID and API key
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To find your Merchant ID and API key:
+
+#. Go to your Viva Wallet account and select the relevant account;
+#. Go to :menuselection:`Settings --> API access` in the navigation menu.
+
+Your Merchant ID and API Key are located in the :guilabel:`Access credentials` section at the top of
+the page. Save the keys and paste them into the Odoo :guilabel:`Merchant ID` and :guilabel:`API Key`
+fields :ref:`when creating a payment method <viva_wallet/method-creation>`.
+
+.. image:: viva_wallet/access-cred.png
+   :alt: merchant ID and API key fields
+
+.. note::
+   These credentials are used for APIs that authenticate with Basic Auth.
+
+.. seealso::
+   `Find your Merchant ID and API Key
+   <https://developer.vivawallet.com/getting-started/find-your-account-credentials/merchant-id-and-api-key/>`_.
+
+.. _viva_wallet/pos-apis:
+
+POS APIs credentials
+~~~~~~~~~~~~~~~~~~~~
+
+To find your POS APIs credentials:
+
+#. Go to your Viva Wallet account and select the relevant account.
+#. Go to :menuselection:`Settings --> API access` in the navigation menu.
+
+Your POS APIs credentials are located in the :guilabel:`POS APIs Credential` section. Click
+:guilabel:`Generate a pair of credentials` to generate them. Save the keys and paste them into the
+Odoo :guilabel:`Client secret` and :guilabel:`Client ID` fields :ref:`when creating a payment method
+<viva_wallet/method-creation>`.
+
+.. image:: viva_wallet/api-cred.png
+   :alt: Client secret and client ID fields
+
+.. warning::
+   These credentials are only displayed once. Ensure you keep a copy to secure them.
+
+.. note::
+   These credentials are used for Android and iOS POS Activation requests, as well as the Cloud
+   Terminal API.
+
+.. seealso::
+   `Find your POS APIs Credentials
+   <https://developer.vivawallet.com/getting-started/find-your-account-credentials/pos-apis-credentials/>`_.
+
+.. _viva_wallet/identifier:
+
+Terminal ID
+~~~~~~~~~~~
+
+To find your terminal ID number:
+
+#. Go to your Viva Wallet account and select the relevant account.
+#. Go to :menuselection:`Sales --> Physical payments --> Card terminals` in the navigation menu.
+
+The terminal ID number is located under the :guilabel:`Terminal ID (TID)` column.
+
+.. image:: viva_wallet/terminal-id.png
+   :alt: Viva terminal ID
+
+.. _viva_wallet/method-creation:
+
+Configure the payment method
+----------------------------
+
+Enable the payment terminal :ref:`in the application settings <configuration/settings>` and
+:doc:`create the related payment method <../../payment_methods>`. Set the journal type as
```

Comment:
I was not able to do this. I didn't see where to enable the payment terminal nor to create the payment method.